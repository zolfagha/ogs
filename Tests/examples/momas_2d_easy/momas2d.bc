#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  C1
 $GEO_TYPE
  PLY_INLET
 $DIS_TYPE
  CONSTANT 4.139268E-12
  
#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  C2
 $GEO_TYPE
  PLY_INLET
 $DIS_TYPE
  CONSTANT 5.837038E-02
  
#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  C3
 $GEO_TYPE
  PLY_INLET
 $DIS_TYPE
  CONSTANT 6.031405E-36
  
#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  C4
 $GEO_TYPE
  PLY_INLET
 $DIS_TYPE
  CONSTANT 2.194295E-107

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  C5
 $GEO_TYPE
  PLY_INLET
 $DIS_TYPE
  CONSTANT 7.000821E-06

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  X1
 $GEO_TYPE
  PLY_INLET
 $DIS_TYPE
  CONSTANT 3.000000E-01

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  X2
 $GEO_TYPE
  PLY_INLET
 $DIS_TYPE
  CONSTANT 2.415886E-01
  
#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  X3
 $GEO_TYPE
  PLY_INLET
 $DIS_TYPE
  CONSTANT 2.416106E-01

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  X4
 $GEO_TYPE
  PLY_INLET
 $DIS_TYPE
  CONSTANT 1.457119E-36
  
#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  S
 $GEO_TYPE
  PLY_INLET
 $DIS_TYPE
  CONSTANT 1.174438E-09
  
#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  CS1
 $GEO_TYPE
  PLY_INLET
 $DIS_TYPE
  CONSTANT 4.001065E-06
  
#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  CS2
 $GEO_TYPE
  PLY_INLET
 $DIS_TYPE
  CONSTANT 1.425364E-53
