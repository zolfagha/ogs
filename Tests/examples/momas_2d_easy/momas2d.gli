#POINTS
0 0 0 0 
1 1 0 0 
2 1.1 0 0 
3 2.1 0 0 
4 2.1 .025 0 
5 1.1 .025 0 
6 1 .025 0 
7 0 .025 0 
#POLYLINE
 $NAME
  PLY_INLET
 $EPSILON
  0.000001
 $POINTS
 0
 7
#POLYLINE
 $NAME
  PLY_OUTLET
 $EPSILON
  0.000001
 $POINTS
 3
 4
#POLYLINE
 $NAME
  PLY_LEFT
 $EPSILON
  0.000001
 $POINTS
 0
 1
 6
 7
 0
#POLYLINE
 $NAME
  PLY_MID
 $EPSILON
  0.000001
 $POINTS
 1
 2
 5
 6
 1 
#POLYLINE
 $NAME
  PLY_RIGHT
 $EPSILON
  0.000001
 $POINTS
 2
 3
 4
 5 
 2
#POLYLINE
 $NAME
  PLY_DOMAIN
 $EPSILON
  0.000001
 $POINTS
 0
 3
 4
 7 
 0
#SURFACE
 $NAME
  SURF_LEFT
 $POLYLINES
  PLY_LEFT
 $TYPE
  0
#SURFACE
 $NAME
  SURF_MID
 $POLYLINES
  PLY_MID
 $TYPE
  0
#SURFACE
 $NAME
  SURF_RIGHT
 $POLYLINES
  PLY_RIGHT
 $TYPE
  0
#SURFACE
 $NAME
  SURF_DOMAIN
 $POLYLINES
  PLY_DOMAIN
 $TYPE
  0
#STOP
