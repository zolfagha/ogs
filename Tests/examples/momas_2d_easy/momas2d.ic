GeoSys-IC  Initial Conditions ------------------------------------------------
#INITIAL_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 1.0
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  C1
 $GEO_TYPE
  PLY_LEFT
 $DIS_TYPE
  CONSTANT 3.850332E-12
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  C2
 $GEO_TYPE
  PLY_LEFT
 $DIS_TYPE
  CONSTANT 6.357587E-14 
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  C3
 $GEO_TYPE
  PLY_LEFT
 $DIS_TYPE
  CONSTANT 1.345836E+00
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  C4
 $GEO_TYPE
  PLY_LEFT
 $DIS_TYPE
  CONSTANT 2.297551E-13

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  C5
 $GEO_TYPE
  PLY_LEFT
 $DIS_TYPE
  CONSTANT 2.332775E-06

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  X1
 $GEO_TYPE
  PLY_LEFT
 $DIS_TYPE
  CONSTANT 1.000000E-06

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  X2
 $GEO_TYPE
  PLY_LEFT
 $DIS_TYPE
  CONSTANT 2.597179E-01
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  X3
 $GEO_TYPE
  PLY_LEFT
 $DIS_TYPE
  CONSTANT 2.447882E-13

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  X4
 $GEO_TYPE
  PLY_LEFT
 $DIS_TYPE
  CONSTANT 3.495376E-01
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  S
 $GEO_TYPE
  PLY_LEFT
 $DIS_TYPE
  CONSTANT 3.907438E-01
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  CS1
 $GEO_TYPE
  PLY_LEFT
 $DIS_TYPE
  CONSTANT 1.675669E-09
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  CS2
 $GEO_TYPE
  PLY_LEFT
 $DIS_TYPE
  CONSTANT 3.046301E-01

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  C1
 $GEO_TYPE
  PLY_MID
 $DIS_TYPE
  CONSTANT 6.615729E-13
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  C2
 $GEO_TYPE
  PLY_MID
 $DIS_TYPE
  CONSTANT 2.916125E-14 
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  C3
 $GEO_TYPE
  PLY_MID
 $DIS_TYPE
  CONSTANT 3.808091E-01
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  C4
 $GEO_TYPE
  PLY_MID
 $DIS_TYPE
  CONSTANT 7.048284E-17

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  C5
 $GEO_TYPE
  PLY_MID
 $DIS_TYPE
  CONSTANT 2.157597E-06

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  X1
 $GEO_TYPE
  PLY_MID
 $DIS_TYPE
  CONSTANT 1.000000E-06

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  X2
 $GEO_TYPE
  PLY_MID
 $DIS_TYPE
  CONSTANT 1.511549E+00
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  X3
 $GEO_TYPE
  PLY_MID
 $DIS_TYPE
  CONSTANT 1.929229E-14

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  X4
 $GEO_TYPE
  PLY_MID
 $DIS_TYPE
  CONSTANT 5.756117E-01
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  S
 $GEO_TYPE
  PLY_MID
 $DIS_TYPE
  CONSTANT 7.912836E+00
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  CS1
 $GEO_TYPE
  PLY_MID
 $DIS_TYPE
  CONSTANT 5.272091E-07
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  CS2
 $GEO_TYPE
  PLY_MID
 $DIS_TYPE
  CONSTANT 1.043583E+00
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  C1
 $GEO_TYPE
  PLY_RIGHT
 $DIS_TYPE
  CONSTANT 3.850332E-12
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  C2
 $GEO_TYPE
  PLY_RIGHT
 $DIS_TYPE
  CONSTANT 6.357587E-14 
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  C3
 $GEO_TYPE
  PLY_RIGHT
 $DIS_TYPE
  CONSTANT 1.345836E+00
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  C4
 $GEO_TYPE
  PLY_RIGHT
 $DIS_TYPE
  CONSTANT 2.297551E-13

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  C5
 $GEO_TYPE
  PLY_RIGHT
 $DIS_TYPE
  CONSTANT 2.332775E-06

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  X1
 $GEO_TYPE
  PLY_RIGHT
 $DIS_TYPE
  CONSTANT 1.000000E-06

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  X2
 $GEO_TYPE
  PLY_RIGHT
 $DIS_TYPE
  CONSTANT 2.597179E-01
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  X3
 $GEO_TYPE
  PLY_RIGHT
 $DIS_TYPE
  CONSTANT 2.447882E-13

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  X4
 $GEO_TYPE
  PLY_RIGHT
 $DIS_TYPE
  CONSTANT 3.495376E-01
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  S
 $GEO_TYPE
  PLY_RIGHT
 $DIS_TYPE
  CONSTANT 3.907438E-01
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  CS1
 $GEO_TYPE
  PLY_RIGHT
 $DIS_TYPE
  CONSTANT 1.675669E-09
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA_REDUCT
 $PRIMARY_VARIABLE 
  CS2
 $GEO_TYPE
  PLY_RIGHT
 $DIS_TYPE
  CONSTANT 3.046301E-01
  
#STOP