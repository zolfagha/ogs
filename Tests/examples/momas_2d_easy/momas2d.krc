GeoSys Kinetic Reactions
#KINREACTIONDATA      
 $SOLVER_TYPE
	1                ; Solver type
 $RELATIVE_ERROR	
	1.e-6		 ; rel. Error
 $MIN_TIMESTEP	
	1.e-6		; min Timestep
 $INITIAL_TIMESTEP	
	1.e-4       ; init Timestep
 $BACTERIACAPACITY
  100.0
 $NO_REACTIONS
   POLYLINE PLY_BC_LEFT

#REACTION ; 
 $NAME
   R1 ;
 $TYPE
   MOB_EQ_REACT;
 $EQUATION
   -1 C1 =  1 X2; 
 $EQ_CONSTANT
 1.0e-12         ; K values

#REACTION ; 
 $NAME
   R2 ;
 $TYPE
   MOB_EQ_REACT;
 $EQUATION
   1 X2 + 1 X3 =  1 C2 ; 
 $EQ_CONSTANT
 1.0            ; K values

#REACTION ; 
 $NAME
   R3 ;
 $TYPE
   MOB_EQ_REACT;
 $EQUATION
   1 X4 =  1 C3 + 1 X2 ;
 $EQ_CONSTANT
 1.0            ; K values

#REACTION ; 
 $NAME
   R4 ;
 $TYPE
   MOB_EQ_REACT;
 $EQUATION
   3 X4 + 1 X3 =  1 C4 + 4 X2 ; 
 $EQ_CONSTANT
 0.1            ; K values
   
#REACTION ; 
 $NAME
   R5 ;
 $TYPE
   MOB_EQ_REACT;
 $EQUATION
   4 X2 + 3 X3 + 1 X4 =  1 C5 ; 
 $EQ_CONSTANT
 1.0e35          ; K values

#REACTION ; 
 $NAME
   R6 ;
 $TYPE
   SORP_EQ_REACT;
 $EQUATION
   3 X2 + 1 X3 + 1 S =  1 CS1 ;
 $EQ_CONSTANT
 1.0e6           ; K values

#REACTION ; 
 $NAME
   R7 ;
 $TYPE
   SORP_EQ_REACT;
 $EQUATION
   2 S + 1 X4 =  1 CS2 + 3 X2 ; 
 $EQ_CONSTANT
 0.1            ; K values
   
#STOP




