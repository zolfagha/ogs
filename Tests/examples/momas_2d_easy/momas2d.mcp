#COMPONENT_PROPERTIES  ; comp0
 $NAME  ; 
  C1 ; 
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0            	; diffusion model type, diffusion constant
  
#COMPONENT_PROPERTIES  ; comp1
 $NAME
  C2 ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0             	; diffusion model type, diffusion constant

#COMPONENT_PROPERTIES  ; comp2
 $NAME
  C3 ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0             	; diffusion model type, diffusion constant

#COMPONENT_PROPERTIES  ; comp3
  $NAME
  C4 ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0            	; diffusion model type, diffusion constant

#COMPONENT_PROPERTIES  ; comp4
  $NAME
  C5 ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  0             	; diffusion model type, diffusion constant
  
#COMPONENT_PROPERTIES  ; comp5
 $NAME  ; 
  X1 ; 
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0            	; diffusion model type, diffusion constant

#COMPONENT_PROPERTIES  ; comp6
 $NAME  ; 
  X2 ; 
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0            	; diffusion model type, diffusion constant
  
#COMPONENT_PROPERTIES  ; comp7
 $NAME
  X3 ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0             	; diffusion model type, diffusion constant

#COMPONENT_PROPERTIES  ; comp8
 $NAME
  X4 ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0             	; diffusion model type, diffusion constant

#COMPONENT_PROPERTIES  ; comp9
  $NAME
  S ;
 $MOBILE
  0; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  0             	; diffusion model type, diffusion constant
  
#COMPONENT_PROPERTIES  ; comp10
  $NAME
  CS1 ;
 $MOBILE
  0; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  0             	; diffusion model type, diffusion constant
  
#COMPONENT_PROPERTIES  ; comp11
  $NAME
  CS2 ;
 $MOBILE
  0; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  0             	; diffusion model type, diffusion constant
 
#STOP









#COMPONENT_PROPERTIES  ; comp0
 $NAME  ; 
  ConsTracer ; 
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  0.0            	; diffusion model type, diffusion constant

; temperary disable the tracer for now
