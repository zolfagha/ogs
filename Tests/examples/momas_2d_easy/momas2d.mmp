GeoSys-MMP  Material Medium Properties ------------------------------------
#MEDIUM_PROPERTIES
 $GEOMETRY_DIMENSION
  2
 $GEOMETRY_AREA
  1.0
 $POROSITY
  1  0.25
 $TORTUOSITY
  1   1.0
 $STORAGE
  1   0.000 
 $PERMEABILITY_TENSOR
  ISOTROPIC  1.0e-2
 $DENSITY
  1  2000.0
 $MASS_DISPERSION
  1  10.0  1.0  
  
#MEDIUM_PROPERTIES
 $GEOMETRY_DIMENSION
  2
 $GEOMETRY_AREA
  1.0
 $POROSITY
  1  0.5
 $TORTUOSITY
  1   1.0
 $STORAGE
  1   0.000 
 $PERMEABILITY_TENSOR
  ISOTROPIC  1.0e-5
 $DENSITY
  1  2000.0
 $MASS_DISPERSION
  1  60.0  6.0  
#STOP
