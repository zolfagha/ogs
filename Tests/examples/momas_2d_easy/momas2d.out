GeoSys-OUT: Output ------------------------------------------------
#OUTPUT // profile
 $NOD_VALUES
  HEAD
  C1
  C2
  C3
  C4
  C5
  X1
  X2
  X3
  X4
  S
  CS1
  CS2
 $ELE_VALUES
  VELOCITY
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  PVD
 $TIM_TYPE
  STEPS 1
#STOP

